configs/json_exporter/config.yml

```diff
@@ -0,0 +1,8 @@
+metrics: []
```

Comment:
As far as I can see it is empty right now. Can you clarify what is the goal to deploy this exporter?